test/unit/test_sap_adt_transaction.py

```
@@ -0,0 +1,418 @@
+    def test_build_report_transaction_creation_content(self):
+        connection = Connection()
+        metadata = sap.adt.ADTCoreData(description='abapgit', language='EN', master_language='EN',
+                                       responsible='DEVELOPER')
+        transaction = sap.adt.Transaction(connection, TRANSACTION_NAME, package='$TMP', metadata=metadata)
+        transaction.definition.transaction_type = 'reportTransaction'
+        transaction.definition.report_transaction = ReportTransactionDefinition(
+            report_name='ZABAPGIT', report_dynnr='1000'
+        )
+
+        content = transaction.definition.to_create_json(transaction.name, transaction.description,
+                                                 transaction.reference.name)
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)